Lib/glyphsLib/builder/guidelines.py

```diff
@@ -36,7 +36,6 @@ def to_ufo_guidelines(self, ufo_obj, glyphs_obj):
         new_guideline = {}
         x, y = guideline.position
         angle = guideline.angle
-        angle = (360 - angle) % 360
```

Comment:
The angle must be between 0 and 360 in UFO.
So you could change the previous line to `angle = guideline.angle % 360`.